grpc/src/main/java/com/replica/api/StreetRouter.java

```diff
@@ -54,14 +54,14 @@ public void routeStreetMode(StreetRouteRequest request, StreamObserver<StreetRou
         GHPoint dest = ghRequest.getPoints().get(1);
 
         StreetRouteReply.Builder replyBuilder = StreetRouteReply.newBuilder();
-        boolean anyPathsFound = false;
+        int pathsFound = 0;
         for (String profile : profilesToQuery) {
             ghRequest.setProfile(profile);
             try {
                 GHResponse ghResponse = graphHopper.route(ghRequest);
                 // ghResponse.hasErrors() means that the router returned no results
                 if (!ghResponse.hasErrors()) {
-                    anyPathsFound = true;
+                    pathsFound += ghResponse.getAll().size();
```

Comment:
Great point - @rregue mentioned yesterday that he and @bnaul were mostly interested in getting a rough estimate of ~how many route options are available on average; if there are on average enough route options available, it would allow us to test changing the approach we use to assign congestion to links (ie route selection) during model runs.

With that in mind, I think the current non-unique count is probably good enough - we know from human testing that maybe 1-2 duplicate routes show up per request, and could adjust the average route count we find with the metric accordingly. But including uniqueness would obviously give us an even clearer picture

I don't want to block this if a uniqueness check might be complex to implement, but how would we feel about a simpler approach? I think even storing route duration + # of links in the route would be enough to use as a "uniqueness key" here (or even just route duration...chances two different routes have different ms durations seems very low)

Reply:
personally I would just do the simplest thing for now, I think it would be enough for us to tune on regardless

Reply:
Cool, gonna merge then and we can add a unique metric in the future if it seems like it would be helpful

Reply:
Yeah I'm definitely on board proceeding with things as written! If we need to track or remove duplicates in the future, we can do it in a follow-up. But FYI I'm not sure route duration would work to check uniqueness since some of the profiles modify the speeds, so I think two profiles could produce the same route with different durations.

The GH responses contain a `PointList` in their `ResponsePath`, and `PointList` implements equals and hashcode. So maybe we could identify duplicates by their `PointList`, at least for non-transit routes.